src/main/java/build/buildfarm/instance/shard/ShardInstance.java

```diff
@@ -855,13 +886,19 @@ public void onError(Throwable t) {
                 } else if (status.getCode() == Code.NOT_FOUND) {
                   casMissCounter.inc();
                   log.log(
-                      Level.FINE, worker + " did not contain " + DigestUtil.toString(blobDigest));
+                      configs.getServer().isEnsureOutputsPresent() ? Level.SEVERE : Level.FINE,
+                      worker + " did not contain " + DigestUtil.toString(blobDigest));
                   // ignore this, the worker will update the backplane eventually
                 } else if (status.getCode() != Code.DEADLINE_EXCEEDED
                     && SHARD_IS_RETRIABLE.test(status)) {
                   // why not, always
                   workers.addLast(worker);
                 } else {
+                  log.log(
+                      configs.getServer().isEnsureOutputsPresent() ? Level.SEVERE : Level.WARNING,
```

Comment:
this is not a severe error, even if we're checking for an AC presence via EOP. SEVERE logs are reserved for bugs or consistency failures.

Reply:
With `bwob` flag enabled this one failure causes whole build to fail, also I believe the EOP flag was added to support bwob feature. So I thought SEVERE might be the better classification.

Reply:
Build failures because of expected behavior or conditions are still not SEVERE. Please change this and the 
log elevation in bytestream to remaing a warning.

Reply:
Reverted log level as per your suggestion.